components/Forms/ComponentFactory.tsx

```diff
@@ -100,15 +75,8 @@ export const ComponentFactory: React.VFC<FactoryProps> = observer(
             {form.fields.map(
               (field: Instance<typeof FormField>, index: number) => {
                 const isChildQuestion =
-                  field.category.key == FieldCategory.PARTNER_DETAILS ||
-                  field.category.key == FieldCategory.SOCIAL_AGREEMENT
-                const styling = isChildQuestion
-                  ? `bg-emphasis px-10 pt-4 ${
-                      field.category.key == FieldCategory.SOCIAL_AGREEMENT
-                        ? ' mb-10'
-                        : ''
-                    }`
-                  : ``
+                  field.category.key == FieldCategory.PARTNER_INFORMATION
+                const styling = isChildQuestion ? 'bg-emphasis px-10 pt-4' : ''
```

Comment:
Does the social agreement question not need a background anymore?

Reply:
I don't think so, we are scrapping it down to only two sections (client/partner) so there isn't really a way to highlight the social agreement question without adding its own section back. If it comes up as a problem I will revert this.